src/contoroller/account/account.test.js

```diff
@@ -0,0 +1,520 @@
+const AccountController = require("./account");
+
+describe("AccountController", () => {
```

Comment:
You can also test the constructor. Create the accountController and test if the params that you pass to the class constructor are set correctly:

```
User = {};
Account = {};
validate = jest.fn();

accountController = new AccountController(Account, User, validate) ;

it('should initialize class members correctly', () => {
  expect(accountController.User).toBe(User);
  expect(accountController.Account).toBe(Account);
  expect(accountController.validateAccount).toBe(validate);
});  
```